client/utils/navigation.js

```diff
@@ -1,11 +1,6 @@
-import { NamedURLResolver } from 'react-router-named-routes'
-import { browserHistory } from 'react-router'
+import { history } from 'base'
 
 // eslint-disable-next-line import/prefer-default-export
-export function navigateTo(routeName) {
-  return browserHistory.push(namedRoute(routeName))
-}
-
-function namedRoute(name, params = {}) {
-  return NamedURLResolver.resolve(name, params)
+export function navigateTo(name) {
+  return history.push({ name })
```

Comment:
But we're not doing that here.  If we leave it as `name`, or even better, rename `name` to something more generic, then we can call `navigateTo` with a richer object as supported by the `useNamedHistory` api, and it will just pass right through.